### PR TITLE
Migrate the project to go modules

### PR DESCRIPTION
**Pull request by krnowak**
This PR migrates the project to the official go module system. It is now not required anymore to put the project in GOPATH.

### CODE REVIEW

**Comment by krnowak** [comment]
Hmph, it looks like I updated the wrong CI file. Should the `.travis.yml` file be dropped then?

**Comment by joaquimrocha** [comment]
I will take a look tomorrow.

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+GO111MODULE=on
+export GO111MODULE
```
Comment (joaquimrocha) [inline Makefile]: nitpick: I assume this would work if we had it only in one line (`export GO11MODULE=on`).

File: .github/workflows/backend.yml [hunk 2]
```diff
@@ -29,22 +29,12 @@ jobs:
...
       run: |
-        cd $RUNNER_WORKSPACE
-        NEBRASKA_REPO=$GITHUB_WORKSPACE
-        export GOPATH=${RUNNER_WORKSPACE}/go
-        NEBRASKA_GO=$GOPATH/src/github.com/$GITHUB_REPOSITORY
-        mkdir -p $GOPATH/bin
-        curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-        cd $NEBRASKA_GO
-        $GOPATH/bin/dep ensure
+        cd $GITHUB_WORKSPACE
```
Comment (joaquimrocha) [inline .github/workflows/backend.yml]: :+1: :+1: :+1:

**Pull request merged**